doc/source/whatsnew/v1.3.3.rst

```diff
@@ -46,7 +46,8 @@ Performance improvements
 Bug fixes
 ~~~~~~~~~
 - Fixed bug in :meth:`.DataFrameGroupBy.agg` and :meth:`.DataFrameGroupBy.transform` with ``engine="numba"`` where ``index`` data was not being correctly passed into ``func`` (:issue:`43133`)
-
+- :meth:`.GroupBy.mean` now supports ``NaT`` values (:issue:`43132`)
```

Comment:
Fixed bug in :meth:\`.GroupBy.mean\` with datetimelike values including \`\`NaT\`\` values returning incorrect results (:issue\`:43132\`)

Reply:
Done, thanks!